content/doc/addons/keycloak.md

```diff
@@ -130,23 +130,57 @@ There are two ways to export Keycloak data:
 
 2. A full export using the environment variables of the Keycloak Java application from the Clever Cloud console.
     - Go to the Keycloak Java application in the Clever Cloud console
-    - In the Java application, you can explicitly Realm you want to export as arguments to  the `CC_EXPORT_REALM` environment variable;
+    - In the Java application, you can explicitly Realms (separated each of them with `,`) you want to export as arguments to the `CC_KEYCLOAK_EXPORT_REALMS` environment variable;
 
     > For instance, the following values export `my_realm` after the application restart:
-        `CC_EXPORT_REALM=my_realm`
+        `CC_KEYCLOAK_EXPORT_REALMS=my_realm`
 
     - restart the Java application from the Clever Cloud console.
 
     > Don't forget to apply change at the bottom of the environment variable page if you use the console.
 
 > This is a total export, including the client's secrets and hashed password
 
-### Importing realms data
+Your exported realm is available in `realms/export` folder in the associated FSBucket.
 
-Uploading previously exported data in the `Import` folder in the associated FSBucket enables importing realms data. The import process starts after restarting the Java application.
+## Importing realms data
```

Comment:
I think we should place this in the `Realms managment` section and move plugins/themes stuff outside of it. 

```suggestion
### Importing realms data
```